queries.sql

```diff
@@ -23,3 +23,40 @@ WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg >= 10.40 AND weight_kg <= 17.30;
+
+/* Begin of a transaction */
+BEGIN;
+
```

Comment:
- [ ] Please from the project requirement, Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that the change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction. 
Please make sure you do that and **Take a screenshot of the results of your actions.** :pray: